keyboards/smithrune/iron160/iron160_h/info.json

```diff
@@ -23,70 +23,70 @@
-                {"label": "}", "matrix": [1, 12], "w": 1, "x": 12.5, "y": 1},
-                {"label": "|", "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1},
-                {"label": "Caps", "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2},
-                {"label": "A", "matrix": [2, 1], "w": 1, "x": 1.75, "y": 2},
-                {"label": "S", "matrix": [2, 2], "w": 1, "x": 2.75, "y": 2},
-                {"label": "D", "matrix": [2, 3], "w": 1, "x": 3.75, "y": 2},
-                {"label": "F", "matrix": [2, 4], "w": 1, "x": 4.75, "y": 2},
-                {"label": "G", "matrix": [2, 5], "w": 1, "x": 5.75, "y": 2},
-                {"label": "H", "matrix": [2, 6], "w": 1, "x": 6.75, "y": 2},
-                {"label": "J", "matrix": [2, 7], "w": 1, "x": 7.75, "y": 2},
-                {"label": "K", "matrix": [2, 8], "w": 1, "x": 8.75, "y": 2},
-                {"label": "L", "matrix": [2, 9], "w": 1, "x": 9.75, "y": 2},
-                {"label": ":", "matrix": [2, 10], "w": 1, "x": 10.75, "y": 2},
-                {"label": "\"", "matrix": [2, 11], "w": 1, "x": 11.75, "y": 2},
-                {"label": "Enter", "matrix": [2, 13], "w": 1.25, "x": 12.75, "y": 2},
-                {"label": "Shift", "matrix": [3, 0], "w": 1.25, "x": 0, "y": 3},
-                {"label": "Z", "matrix": [3, 2], "w": 1, "x": 2.25, "y": 3},
-                {"label": "X", "matrix": [3, 3], "w": 1, "x": 3.25, "y": 3},
-                {"label": "C", "matrix": [3, 4], "w": 1, "x": 4.25, "y": 3},
-                {"label": "V", "matrix": [3, 5], "w": 1, "x": 5.25, "y": 3},
-                {"label": "B", "matrix": [3, 6], "w": 1, "x": 6.25, "y": 3},
-                {"label": "N", "matrix": [3, 7], "w": 1, "x": 7.25, "y": 3},
-                {"label": "M", "matrix": [3, 8], "w": 1, "x": 8.25, "y": 3},
-                {"label": "<", "matrix": [3, 9], "w": 1, "x": 9.25, "y": 3},
-                {"label": ">", "matrix": [3, 10], "w": 1, "x": 10.25, "y": 3},
-                {"label": "?", "matrix": [3, 11], "w": 1, "x": 11.25, "y": 3},
-                {"label": "Shift", "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3},
-                {"label": "MO(1)", "matrix": [3, 13], "w": 1, "x": 14, "y": 3},
-                {"label": "Ctrl", "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4},
-                {"label": "Win", "matrix": [4, 1], "w": 1, "x": 1.5, "y": 4},
-                {"label": "Alt", "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4},
-                {"label": "Space", "matrix": [4, 6], "w": 7, "x": 4, "y": 4},
-                {"label": "Alt", "matrix": [4, 10], "w": 1.5, "x": 11, "y": 4},
-                {"label": "Win", "matrix": [4, 11], "w": 1, "x": 12.5, "y": 4},
-                {"label": "Ctrl", "matrix": [4, 12], "w": 1.5, "x": 13.5, "y": 4}
-            ]
+                "layout": [
+                    { "label": "Esc",   "matrix": [0, 0 ],            "x": 0,     "y": 0 },
```

Comment:
Please don't do this. Run `qmk format-json` on this file.

Reply:
Sorry, I'm not sure I understand what I see, do what exactly?

Reply:
The formatting you've applied to the layouts is unnecessary, and will be undone when running `qmk format-json` - this is likely to be automated in future.